tests/python_tests/test_whisper_generate_api.py

```diff
@@ -515,12 +524,21 @@ def test_longform_audio_return_timestamps_en(model_descr, test_sample):
         return_timestamps=True,
     )
 
+    streamer_result = ""
+
+    def streamer(word: str) -> bool:
+        nonlocal streamer_result
+        streamer_result += word
+        return False
+
     genai_result = pipe.generate(
         test_sample,
         return_timestamps=True,
+        streamer=streamer,
```

Comment:
Here's a simpler way to implement streamer. https://github.com/openvinotoolkit/openvino.genai/blob/1fb7cafa0120eab72c5cd4bebd88f345f9e7901d/tests/python_tests/test_generate_api.py#L695 Could you adapt it here for string concatenation

Reply:
Done. We would probably need to explicitly add `None` as a return type of streamer callback. Currently it is just `bool`. Once `.pyi` will be generated there will be lint warning here.

Reply:
You can probably define streamer as `lambda x: res_str.append(x) or False` in that case